pulpcore.te

```diff
@@ -9,58 +9,138 @@ type pulpcore_t;
-allow pulpcore_t self:unix_stream_socket create_stream_socket_perms;
-allow pulpcore_t self:netlink_route_socket { create_socket_perms nlmsg_read };
-allow pulpcore_t self:tcp_socket create_stream_socket_perms;
-allow pulpcore_t self:udp_socket create_stream_socket_perms;
+allow pulpcore_t pulpcore_server_exec_t:file getattr;
+allow pulpcore_server_t pulpcore_exec_t:file getattr;
+
+allow pulpcore_server_t self:fifo_file rw_fifo_file_perms;
+allow pulpcore_server_t self:unix_stream_socket create_stream_socket_perms;
+allow pulpcore_server_t self:netlink_route_socket { create_socket_perms nlmsg_read };
+allow pulpcore_server_t self:tcp_socket create_stream_socket_perms;
+allow pulpcore_server_t self:udp_socket create_stream_socket_perms;
+
+# /etc
+read_files_pattern(pulpcore_t, pulpcore_etc_t, pulpcore_etc_t)
+read_files_pattern(pulpcore_server_t, pulpcore_etc_t, pulpcore_etc_t)
 
+# /var/lib
 manage_dirs_pattern(pulpcore_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
 manage_files_pattern(pulpcore_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
-relabel_files_pattern(pulpcore_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
-manage_lnk_files_pattern(pulpcore_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
-files_var_lib_filetrans(pulpcore_t, pulpcore_var_lib_t, { dir file })
 
+manage_dirs_pattern(pulpcore_server_t, pulpcore_server_var_lib_t, pulpcore_server_var_lib_t)
+manage_files_pattern(pulpcore_server_t, pulpcore_server_var_lib_t, pulpcore_server_var_lib_t)
+ 
+manage_dirs_pattern(pulpcore_t, pulpcore_server_var_lib_t, pulpcore_server_var_lib_t)
+manage_files_pattern(pulpcore_t, pulpcore_server_var_lib_t, pulpcore_server_var_lib_t)
+
+filetrans_pattern(pulpcore_server_t, pulpcore_var_lib_t, pulpcore_server_var_lib_t, { dir file })
+manage_dirs_pattern(pulpcore_server_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
+manage_files_pattern(pulpcore_server_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
+read_lnk_files_pattern(pulpcore_server_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
+write_sock_files_pattern(pulpcore_t, pulpcore_var_lib_t, pulpcore_var_lib_t)
+
+# /run
 manage_dirs_pattern(pulpcore_t, pulpcore_var_run_t, pulpcore_var_run_t)
 manage_files_pattern(pulpcore_t, pulpcore_var_run_t, pulpcore_var_run_t)
-files_pid_filetrans(pulpcore_t, pulpcore_var_run_t, { dir })
+files_pid_filetrans(pulpcore_t, pulpcore_var_run_t, { file dir })
 
+manage_dirs_pattern(pulpcore_server_t, pulpcore_server_var_run_t, pulpcore_server_var_run_t)
+manage_files_pattern(pulpcore_server_t, pulpcore_server_var_run_t, pulpcore_server_var_run_t)
+files_pid_filetrans(pulpcore_server_t, pulpcore_server_var_run_t, { file dir })
+
+# /tmp
 manage_dirs_pattern(pulpcore_t, pulpcore_tmp_t, pulpcore_tmp_t)
 manage_files_pattern(pulpcore_t, pulpcore_tmp_t, pulpcore_tmp_t)
-relabel_files_pattern(pulpcore_t, pulpcore_tmp_t, pulpcore_tmp_t)
+manage_lnk_files_pattern(pulpcore_t, pulpcore_tmp_t, pulpcore_tmp_t)
+#relabel_files_pattern(pulpcore_t, pulpcore_tmp_t, pulpcore_tmp_t)
```

Comment:
I have issues with commented code. Is this needed or can it be removed?

Reply:
A result of testing, no need for commented code.